handlers/assign/assign.go

```diff
@@ -14,6 +14,204 @@ const (
 	Unassign = "/unassign"
 )
 
+var (
+	assignRe = regexp.MustCompile(`(?mi)^/(un)?assign(( @?[-\w]+?)*)\s*$`)
+	// CCRegexp parses and validates /cc commands, also used by blunderbuss
+	CCRegexp = regexp.MustCompile(`(?mi)^/(un)?cc(( +@?[-/\w]+?)*)\s*$`)
```

Comment:
`CCRegexp` -> `ccRegexp` ?

Reply:
> `CCRegexp` -> `ccRegexp` ?

CCRegexp var is used by other packages, if i make it lower case, other packges cannot access this variable.

Reply:
Okay, Just keep the variable name consistent with `assignRe` or `AssignRegExp` and also consider `CCRegexp` in `handlers/util.go`.

Reply:
So, CCRegexp -> CCRegExp ?

Reply:
OK